docs/semgrep-ci.md

```diff
@@ -0,0 +1,60 @@
+# Semgrep CI
```

Comment:
This is the only new copy @pabloest the other files move things around

Reply:
Thanks for pointing this out :)